src/style.css

```diff
@@ -1,3 +1,132 @@
 body {
   margin: 0;
+  display: block;
+  padding: 0;
+  box-sizing: border-box;
+
+  overflow-x: hidden;
+}
+
+.header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  margin: 0 auto;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  top: 10px;
+  left: 50px;
```

Comment:
Avoid using absolute positioning for elements like `.logo` (lines 25 and 26) unless necessary. This can make the layout less flexible and harder to maintain. Consider using flexbox or grid properties to align and place elements in the header.